src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -504,6 +505,10 @@
                     @if {@defaultMutableValues(optionalParams)}
                         {@defaultMutableValues(optionalParams)}
                     @end
+                    @if oneOfParams
+                      {@checkOneOfParams(oneOfParams)}
```

Comment:
This is using two space indentation, whereas it looks like everything else is four space.